prac_05/hex_colours.py

```diff
@@ -0,0 +1,28 @@
+COLORS = {
+    "Absolute Zero": "#0048ba",
+    "Amethyst": "#9966cc",
+    "Aquamarine2": "#76eec6",
+    "BlueViolet":"#8a2be2",
+    "Baby Blue":"#89cff0",
+    "Beige":"#f5f5dc",
+    "Boysenberry":"#873260",
+    "Brandeis Blue": "#0070ff",
+    "Brass": "#b5a642",
+    "BrickRed":"#cb4154"}
+
+def main():
+    color_name = validate_color_name()
```

Comment:
Better to add docstring.

Reply:
Added the docstring comment for the main function.